test/fixtures/unit.js

```diff
@@ -11,6 +11,27 @@ function copyErrors (testEnd) {
   return testEnd;
 }
 
+/**
+ * Creates an object that has a cyclical reference.
+ */
+function createCyclical () {
+  const cyclical = {};
+  cyclical.cycle = cyclical;
+  return cyclical;
+}
+
+/**
+ * Creates an object that references another object more
+ * than once in an acyclical way.
+ */
+function createDuplicateAcyclical () {
+  const duplicate = {};
```

Comment:
To improve code coverage a little bit, and to weed out errors that result in objects rendering as empty without keys, perhaps include at least one key-value pair in here.

Reply:
Good point. Added keys to each of the objects so we can identify them in the output.